docs/source/connecting_datasource.rst

```diff
@@ -0,0 +1,90 @@
+Data Source
+===========
+
+It is possible to connect a cloud provider to Neomaril. Currently, Neomaril supports the following providers:
```

Comment:
I think it is important here to make clear that you will could use this cloud provider data sources as data entrance at training experiments. Also @gkardoso do we already have this adaptation to enter the dataset_hash here? I didn't saw at the code. We can create a issue to make it :smile:

Reply:
What do you think about:
"It is possible to connect a cloud provider to Neomaril. The cloud provider is a source of data, where you can store and get the data that is saved data to create/perform models.

Currently, Neomaril supports the following providers:..."?

Reply:
[Resolved](https://github.com/datarisk-io/mlops-neomaril-codex/pull/40/commits/8e091cec28681f5374f809e36a78d2e02a7d8929) (I think)

Reply:
Not exactly... I will talke to Gui and checki it :smile:

Reply:
This is done!